frontend/src/pages/LogsPage/LogsTable/LogsTable.tsx

```diff
@@ -357,18 +357,18 @@ export const findMatchingLogAttributes = (
 		}
 
 		let matchingAttribute: string | undefined = undefined
-		if (bodyQueryValue && value.indexOf(bodyQueryValue) !== -1) {
+		if (
+			bodyQueryValue &&
+			key === 'message' &&
```

Comment:
Use the graphql enum here:
https://github.com/highlight/highlight/blob/b3bb71fb7d62a108513e03518c05af0d345faffc/backend/private-graph/graph/schema.graphqls#L555

Reply:
And is `bodyQueryValue` matching on the wrong thing? The backend remaps `body` -> `message` so `message` is what the frontend should be matching on.

Reply:
I think it's matching on the right thing, but we're using two different names for it, which is confusing. I'm calling it `body` but we have to match against the `message` attribute.